README.md

```diff
@@ -1,24 +1,43 @@
 # Mobility Data Specification
 
-A data standard and API specification for *mobility as a service* providers, such as Dockless Bikeshare, E-Scooters, and Shared Ride providers who work within the public right of way.
+In October of 2018, the City of Los Angeles implemented its first set of data specifications and data sharing requirements focused on dockless e-scooters and bicycles.
 
-Inspired by [GTFS](https://developers.google.com/transit/gtfs/reference/) and [GBFS](https://github.com/NABSA/gbfs). Specifically, the goals of the Mobility Data Specification (**MDS**) are to provide API and data standards for municipalities to help ingest, compare and analyze *mobility as a service* provider data. 
+The Mobility Data Specification (MDS) helps the City ingest and analyze information from for-profit companies who operate dockless scooters and bicycles in the public right-of-way. MDS is a key piece of digital infrastructure that helps the Los Angeles Department of Transportation (LADOT) understand how dockless mobility operates in Los Angeles, which then helps LADOT carry out its core responsibilities of safety, equity, and congestion relief.
 
-The specification is a way to implement realtime data sharing, measurement and regulation for municipalities and *mobility as a service* providers. It is meant to ensure that governments have the ability to enforce, evaluate and manage providers. 
+Mobility providers are required to share data with LADOT as part of the City of Los Angeles&#39; Dockless Mobility Permit. Standardizing data collection between different providers improves cooperation, innovation, and efficiency of the City&#39;s transportation network.
 
-**MDS** is currently comprised of two distinct components:
+**MDS Components** 
 
-* The [`provider`][provider] API is to be implemented by *mobility as a service* providers, for data exchange and operational information that a municipality will query. `provider` presents the historical view of operations.
+* The [`provider`][provider] Application Program Interface (API) was created in May 2018 to be implemented by mobility providers. When a municipality queries information from a mobility provider, the Provider API automatically generates a historical view of operations in a standard format.
 
-* The [`agency`][agency] API is to be implemented by *municipalities* and other regulatory agencies, for providers to query and integrate with during operations. `agency` provides tools to inform and permit future operations.
+* The [`agency`][agency] API was created in April 2019 to be implemented by regulatory agencies. Providers query the Agency API when an event occurs, like a trip starting or ending.
```

Comment:
Hm, I wouldn't say it was created in April of 2019.  It was conceived in the same time frame as Provider.  It did go through some revisions, and it wasn't implemented by LADOT until February of 2019.  April 2019 is when the reference implementation went into production.

Reply:
@karcass @jpayne00 how does 

`The [`agency`][agency] API is implemented by the regulatory agency, ie,  LADOT or the County of Los Angeles. Providers query the API during operations to get information from the regulatory agency.`